### PR TITLE
Migration to AndroidX

### PR DESCRIPTION
**Pull request by RingerJK**

### CODE REVIEW

File: gradle/versions.gradle [hunk 1]
```diff
@@ -6,7 +6,8 @@ ext {
     compile_sdk_version = 28
     build_tools_version = '28.0.3'
 
-    support_lib = '27.1.1'
+    android_x_annotation = '1.1.0-alpha01'
```
Comment (yunikkk) [inline gradle/versions.gradle]: Alpha versions? Think this should be risky to use them, are there 1.0.x releases?

Comment (RingerJK) [reply gradle/versions.gradle]: yep, changed to stable

**Comment by RingerJK** [comment]
@yunikkk PTAL

File: gradle/versions.gradle [hunk 2]
```diff
@@ -6,7 +6,8 @@ ext {
     compile_sdk_version = 28
     build_tools_version = '28.0.3'
 
-    support_lib = '27.1.1'
+    android_x_annotation = '1.0.2'
```
Comment (yunikkk) [inline gradle/versions.gradle]: Save just one variable `android_x`?

Comment (RingerJK) [reply gradle/versions.gradle]: I'm not sure that they're compatible [https://developer.android.com/jetpack/androidx/releases](url) Let's keep separate versions for now

Comment (yunikkk) [reply gradle/versions.gradle]: OK

**Comment by yunikkk** [comment]
Does `Examples/build.gradle` need to be updated, too?

**Comment by RingerJK** [comment]
@yunikkk pr was updated. PTAL

**Pull request merged**